### PR TITLE
[CSBS | MRS] Type fixes

### PR DESCRIPTION
**Pull request by Aloento**
Fix tests and type errors.

### CODE REVIEW

File: openstack/csbs/v1/backup/testing/requests_test.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func TestQueryResourceCapability(t *testing.T) {
 			_, _ = fmt.Fprint(w, queryResponse)
 		})
 
-	options := resource.ResourceBackupCapOpts{CheckProtectable: []resource.ResourceCapQueryParams{
+	options := resource.GetResBackupOpts{CheckProtectable: []resource.ResourceCapQueryParams{
```
Comment (anton-sidelnikov) [inline openstack/csbs/v1/backup/testing/requests_test.go]: please revert

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: openstack/csbs/v1/backup/testing/requests_test.go (hunk 1 above)
Comment (Aloento) [reply openstack/csbs/v1/backup/testing/requests_test.go]: done

**Comment by Aloento** [comment]
recheck

**Review by anton-sidelnikov — APPROVED** [review]

**Review by artem-lifshits — APPROVED** [review]

**Comment by anton-sidelnikov** [comment]
regate

**Pull request merged**